app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -3,358 +3,176 @@ class PipelineSampleReport extends React.Component {
-                        <i className='fa fa-caret-down'></i>
-                      </div>
-                    </th> : '' }
-                    { (this.view_level === 'species') ?
-                    <th>
-                     NT Species e
-                      <div className='sort-controls left'>
-                        <i className='fa fa-caret-up'></i>
-                        <i className='fa fa-caret-down'></i>
-                      </div>
-                    </th> : '' }
-
-
-                    { (this.view_level === 'species') ?
-                    <th>NR Species Z
-                      <div className='sort-controls left'>
-                        <i onClick={ this.columnSorting } className={ `${this.getActiveSort('lowest_species_nr_zscore')} fa fa-caret-up sort_by=lowest_species_nr_zscore` }></i>
-                        <i onClick={ this.columnSorting } className={ `${this.getActiveSort('highest_species_nr_zscore')} fa fa-caret-down sort_by=highest_species_nr_zscore` }></i>
-                      </div>
-                    </th> : '' }
-                    { (this.view_level === 'species') ?
-                    <th>NR Species rM
-                      <div className='sort-controls left'>
-                        <i onClick={ this.columnSorting } className={ `${this.getActiveSort('lowest_species_nr_rpm')} fa fa-caret-up sort_by=lowest_species_nr_rpm` }></i>
-                        <i onClick={ this.columnSorting } className={ `${this.getActiveSort('highest_species_nr_rpm')} fa fa-caret-down sort_by=highest_species_nr_rpm` }></i>
-                      </div>
-                    </th> : '' }
-
-                    { (this.view_level === 'species') ?
-                    <th>NR Species r
-                      <div className='sort-controls left'>
-                        <i className='fa fa-caret-up'></i>
-                        <i className='fa fa-caret-down'></i>
-                      </div>
-                    </th> : '' }
- 
-                    { (this.view_level === 'species') ?
-                    <th>Aggregate Score
-                      <div className='sort-controls left'>
-                        <i className='fa fa-caret-up'></i>
-                        <i className='fa fa-caret-down'></i>
-                      </div>
-                    </th> : '' }
-
+                    <th>Taxonomy</th>
+                    { this.render_column_header('NT+NR', 'ZZRPM',  'nt_aggregatescore') }
+                    { this.render_column_header('NT', 'Z',   'nt_zscore') }
+                    { this.render_column_header('NT', 'rPM', 'nt_rpm')    }
+                    { this.render_column_header('NT', 'r',   'nt_r')      }
+                    { this.render_column_header('NT', '%id', 'nt_percentidentity')    }
+                    { this.render_column_header('NT', 'AL',   'nt_alignmentlength')    }
+                    { this.render_column_header('NT', 'Lg1/E',  'nt_neglogevalue')    }
```

Comment:
log(1/E) [less cryptic?]

Reply:
Interesting.   I thought `log` meant natural logarithm (base e), same as `ln`, whereas `lg` meant log base 10.   That definition, apparently, is not as widely accepted as I had thought.  http://mathworld.wolfram.com/Lg.html